reports/Arizona.md

```diff
@@ -11,3 +11,33 @@ id: az-phoenix-1
 **Links**
 
 * https://twitter.com/PhxProtestLIVE/status/1268390978571825156?s=20
+
+### Officers arrest dozens of people, fail to exhibit probable cause | May 30th
+
+Officers arrested 114 people on Saturday, May 30, most for rioting. The probable cause statements submitted to the court by police justifying the arrests of many defendants were identical from one defendant to another, and alleged only that a crowd of people had engaged in criminal activity, without alleging that the particular defendant arrested had engaged in criminal activity.
+
+tags: arrest
+
+id: az-phoenix-2
+
+**Links**
+
+* [Local news coverage](https://www.abc15.com/news/local-news/investigations/phoenix-police-arrests-dozens-with-copy-and-paste-evidence)
```

Comment:
I noticed in the article this specific excerpt: "During her initial appearance, a judge stated her arrest didn’t have sufficient evidence. However, she was still held and transferred to ICE custody. "

I think this is important to include in the description - mass-arresting protesters and turning them over to ICE is deeply concerning

And also this: "Attorneys said most of the protesters were charged with rioting, which is a class 5 felony. "

Mass arresting for "disorderly conduct" is shitty, but felonies can fuck you up.

I would also maybe include the "abuse-of-power" or "inhumane-treatment" tags